action.yml

```diff
@@ -23,6 +23,16 @@ description: |
           ecr_repository: 'demo/example'
           python-version: 3.10
   ```
+
+  ## Locating Container Images
+
+  Python Docker images are located in AWS ECR, V-Prod. Their URI looks like the below snippet.
+
+  ```yaml
+          064859874041.dkr.ecr.us-east-2.amazonaws.com/<octopusgroup>/<app name>
```

Comment:
Remove this indent in code-block from all PRs.
No need for it to be specified as yaml, you can use plain code-block.

Reply:
Changed yaml to text, removed indent